core/kubernetes/main/openhim/volume/openhim-console/default.json

```
@@ -2,9 +2,9 @@
   "version": "1.13.0",
   "minimumCoreVersion": "5.2.0",
   "protocol": "https",
-  "host": "openhim-core.api.instant",
+  "host": "192.168.99.100",
   "hostPath": "",
-  "port": 443,
+  "port": 30604,
```

@bradsawadye Same as above

```suggestion
  "port": 8082,
```